### PR TITLE
Publishing version 3.24.214.0 of Ditto

### PR DESCRIPTION
**Pull request by sabrogden**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/27428)

### CODE REVIEW

File: manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+- Ditto Clipboard
+InstallerType: exe
+ShortDescription: Ditto is an extension to the standard windows clipboard. It saves each item placed on the clipboard allowing you access to any of those items at a later time.
+PackageUrl: https://ditto-cp.sourceforge.io/
+LicenseUrl: https://tldrlegal.com/license/gnu-general-public-license-v3-(gpl-3)
+Installers:
+- Architecture: x86
+  InstallerUrl: https://github.com/sabrogden/Ditto/releases/download/3.24.214.0/DittoSetup_3_24_214_0.exe
+  InstallerSha256: dc92b7e380a6339fee25a145fb7325b7d4f0591c792c9dd0a3e39a293c26f6ae
+  InstallerSwitches:
```
Comment (jedieaston) [inline manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml:17]:
```suggestion
```

File: manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    Silent: /VERYSILENT /SUPPRESSMSGBOXES /NORESTART /CLOSEAPPLICATIONS
```
Comment (jedieaston) [inline manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml:18]:
```suggestion
```

File: manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+    SilentWithProgress: /SILENT /SUPPRESSMSGBOXES /NORESTART /CLOSEAPPLICATIONS
```
Comment (jedieaston) [inline manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml:19]:
```suggestion
```

File: manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml [hunk 4]
```diff
@@ -0,0 +1,22 @@
+PackageIdentifier: Ditto.Ditto
+PackageVersion: 3.24.214.0
+PackageName: Ditto
+Publisher: Scott Brogden
+Moniker: Ditto
+License: GNU General Public License version 3.0 (GPLv3)
+Tags:
+- Ditto Clipboard
+InstallerType: exe
```
Comment (jedieaston) [inline manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml:9]:
```suggestion
InstallerType: inno
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Since Ditto uses Inno Setup for its installer, you don't need to list the silent install switches. If the installer type is "inno", winget knows what to do. Once you make these changes, you should be set!

Edit: Sorry it got split into two reviews.

File: manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml (hunk 3 above)
Comment (jedieaston) [inline manifests/d/Ditto/Ditto/3.24.214.0/Ditto.Ditto.yaml:19]:
```suggestion
```

**Review by jedieaston — CHANGES_REQUESTED** [review]
Now just set the InstallerType to “inno” instead of “exe” and you’re good :)

**Comment by quhxl** [comment]
Your manifest is not wrong, anyway I submitted #27443 to not remove all metadata inserted in #24081. Have a look at wingetcreate, with it you can create manifests very easily and previous specifications are taken over.

**Pull request closed without merging**